tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml

```diff
@@ -290,6 +290,85 @@ TestOnboarderAccessParticipantDirectoryandFilter:
       Id: $PAGE.care_platform_participant.status_dropdown_options$
     - Type: click
       Strategy: xpath
-      Id: $PAGE.care_platform_participant.status_dropdown_options$
-      
+      Id: $PAGE.care_platform_participant.status_dropdown_options$      
 
+# CXTB-256: Click and View Participant Record from Listing
```

Comment:
We usually use the TC number here.
I believe the one corresponding to this is CXTB-145

Reply:
already fixed.